analysis/grammar_search_analysis.py

```diff
@@ -88,6 +116,8 @@ def _run_proxy_analysis_for_env(env_name: str,
         "timeout": 1,
         "make_videos": False,
         "grammar_search_max_predicates": 50,
+        "grammar_search_size_weight": 0.,
```

Comment:
note that this can cause disparity from full pipeline, but still okay for the sake of easier investigating